Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
+
+2. Generate a Personal Access Token (PAT) for your Azure DevOps Organization. When generating the Personal Access Token (PAT), assign the following scopes:
+
+>a. Agent Pools - Read & Manage
+>b. Deployment Groups - Read & Manage
+
+3. Create a service principle and add it to your subscription as Owner. This is the subscription that will be used to host the image and Agent VM.
```

Comment:
principal not principle, fix the typo please.

Let's add a link showing how to create a service principal, a Microsoft doc.

Reply:
Corrected the typo and added link to instructions.